.github/workflows/tests.yml

```diff
@@ -110,6 +110,38 @@ jobs:
         uses: huggingface/hf-workflows/.github/actions/post-slack@main
         with:
           slack_channel: ${{ env.CI_SLACK_CHANNEL }}
-          title: 🤗 Results of the TRL CI with dev dependencies
+          title: Results with ${{ matrix.python-version }} on ${{ matrix.os }} without optional dependencies
           status: ${{ job.status }}
           slack_token: ${{ secrets.SLACK_CIFEEDBACK_BOT_TOKEN }}
+
+  tests_min_versions:
+    name: Tests with minimum versions
+    runs-on: 'ubuntu-latest'
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Python 3.12
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.12'
+          cache: "pip"
+          cache-dependency-path: |
+              setup.py
+              requirements.txt
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install accelerate==0.34.0
+          python -m pip install datasets==2.21.0
+          python -m pip install transformers==4.46.0
```

Comment:
For my understanding, if we bump the min version in `setup.py` we also need to bump it here as well right?

Reply:
That's right. 
Ideally, we would be able to pip install minimal versions via, eg, `pip install --prefer_min .`, but it's not possible yet, see https://github.com/pypa/pip/issues/8085